<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemManager</class>
 <widget class="QDialog" name="SystemManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tool/icons/Images/config.png</normaloff>:/tool/icons/Images/config.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5" rowminimumheight="0,43">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout" columnminimumwidth="110,0,216">
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QPushButton" name="btnCancel">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>4</y>
            <width>100</width>
            <height>35</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel(&amp;C)</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnOk">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>4</y>
            <width>100</width>
            <height>35</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Ok(&amp;O)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <widget class="QPushButton" name="btnUpdate">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>4</y>
            <width>100</width>
            <height>35</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Update(&amp;U)</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabPlugin_Sort">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabPluginLst">
        <attribute name="title">
         <string>System module info</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tablePluginLst">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="autoScroll">
            <bool>true</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="rowCount">
            <number>26</number>
           </property>
           <property name="columnCount">
            <number>8</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabOrder">
        <attribute name="title">
         <string>System module ordering</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3" columnminimumwidth="0,80">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="0" column="1">
            <widget class="QWidget" name="widget_3" native="true">
             <widget class="QPushButton" name="btnOrigPluginMoveDown">
              <property name="geometry">
               <rect>
                <x>3</x>
                <y>296</y>
                <width>71</width>
                <height>282</height>
               </rect>
              </property>
              <property name="text">
               <string>DOWN</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btnOrigPluginMoveUp">
              <property name="geometry">
               <rect>
                <x>3</x>
                <y>-1</y>
                <width>71</width>
                <height>282</height>
               </rect>
              </property>
              <property name="text">
               <string>UP</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableOrigPluginsSort">
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>true</bool>
             </property>
             <property name="gridStyle">
              <enum>Qt::SolidLine</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>26</number>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>60</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>18</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>18</number>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column/>
             <column/>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Utility</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>System ID:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>System Name:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtSystemID">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtSystemName">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkEnableHotPlug">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Enable Hot Plug</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
